setup.py

```diff
@@ -122,6 +122,17 @@
     author_email='rbean@redhat.com',
     url='https://github.com/fedora-infra/fedmsg/',
     license='LGPLv2+',
+    classifiers=[  # https://pypi.python.org/pypi?%3Aaction=list_classifiers
+        'Development Status :: 3 - Alpha',
```

Comment:
After some discussion on IRC, there appears to be some disagreement on this particular classifier. I'd like expand upon my justification. Those who don't agree with my justification, please make a counter-proposal. Let's have a discussion about this.

All these justifications assume that fedmsg follows semantic versioning 2.0.0 (http://semver.org/). @threebean, since you're the original author, is this assumption false?

As stated above, it's currently releasing under the a 0.y.z scheme which means the API is not stable and could change at any time.

Secondly, there are several API changes I would like to enact prior to declaring fedmsg's API as stable. These include (but are not necessarily limited to): 

1. Proper handling of text vs binary data. This is particularly important since ZeroMQ deals with binary data, not text. Some APIs should start requiring ``unicode``/``str`` objects and rejecting binary data (e.g. the ``fedmsg.publish`` API) and some APIs should start returning unicode objects.

2. I'd prefer to drop moksha and move the functionality we need into fedmsg proper.

3. I'd like to move away from a Python file format for configuration.

4. I'd like to change the on-the-wire format of fedmsgs. I believe the documentation includes using pyzmq directly. We need to decide whether or not this will be a public API.

5. I'd like to unify the CLI with click.

All that to say, I'd like to take advantage of the alpha status of the API.